ang/civicase/dashboard/services/add-case-dashboard-action-button.service.js

```diff
@@ -7,11 +7,11 @@
    * Dashboard Action Button for adding new cases.
    *
    * @param {object} $location the location service.
```

Comment:
We should remove `$location` as its not used anymore?

Reply:
The `$location` service is still used to determine on which dashboard we currently are (cases, awards, prospects, etc.):
https://github.com/compucorp/uk.co.compucorp.civicase/blob/ae0ef4a0d3f30a9efa474f89dad4877958271b4b/ang/civicase/dashboard/services/add-case-dashboard-action-button.service.js#L36-L40